CMakeLists.txt

```diff
@@ -210,15 +240,22 @@ macro(xeus_r_create_target target_name linkage output_name)
         set(XEUS_R_XEUS_TARGET xeus-static)
     endif ()
 
-    target_link_libraries(${target_name} PUBLIC ${XEUS_R_XEUS_TARGET})
+    if (EMSCRIPTEN)
```

Comment:
Not related to this change, but I don't see how the "regular" (i.e. not wasm) kernel can work if it does not link with xeus-zmq instead of xeus.